src/client/power-pages/create/Contentsnippet.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as nls from "vscode-nls";
+nls.config({
+    messageFormat: nls.MessageFormat.bundle,
+    bundleFormat: nls.BundleFormat.standalone,
+})();
+// const localize: nls.LocalizeFunc = nls.loadMessageBundle();
+import * as vscode from "vscode";
+import {
+    createFileWatcher,
+    createRecord,
+    formatFileName,
+    formatFolderName,
+    isNullOrEmpty,
+} from "./utils/CommonUtils";
+import { QuickPickItem } from "vscode";
+import { MultiStepInput } from "./utils/MultiStepInput";
+import path from "path";
+import { statSync } from "fs";
+import { Tables, YoSubGenerator } from "./constants";
+
+export const createContentSnippet = async (
+    context: vscode.ExtensionContext,
+    selectedWorkspaceFolder: string | undefined,
+    yoGenPath: string | null
+): Promise<void> => {
+    try {
+        if (selectedWorkspaceFolder) {
```

Comment:
General guidelines: 
Try to put less if-else condition. It's not a good programming practice. Moreover your code looks complicated.

if (!selectedWorkspaceFolder){
return}
....

If you follow the above pattern then it becomes readable

Reply:
Thanks, will keep it in mind.